pyproject.toml

```diff
@@ -14,18 +14,16 @@ python-dotenv = "^1.0.0"
 cognite-sdk = {version = ">=7.5.4, <7.6", extras = ["pandas"]}
 pandas = "^2.1.1"
 pyyaml = "^6.0.1"
-dacite = "^1.8.1"
-regex = "^2023.6.3"
```

Comment:
@gregertw Do you know why these are here? 

`dacite` is for creating `dataclasses` from a `dict`

and `regex` is a extension of the `builtin` module

Reply:
Old stuff from data-model-examples when we had to do more that is now in the SDK.